data-import-from-s3/src/main/java/no/unit/nva/dataimport/S3Reader.java

```diff
@@ -0,0 +1,85 @@
+package no.unit.nva.dataimport;
+
+import com.amazonaws.services.dynamodbv2.document.Item;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import no.unit.nva.publication.storage.model.DatabaseConstants;
+import no.unit.nva.s3.S3Driver;
+import nva.commons.core.JsonUtils;
+import software.amazon.ion.IonReader;
+import software.amazon.ion.IonWriter;
+import software.amazon.ion.system.IonReaderBuilder;
+import software.amazon.ion.system.IonTextWriterBuilder;
+
+class S3Reader {
+
+    public static final String PK0 = DatabaseConstants.PRIMARY_KEY_PARTITION_KEY_NAME;
+    // Looking for Strings '<end_of_previous_object>}<possible_white_space>{"Item":{"PK0"'
+    public static final String CONSECUTIVE_JSON_OBJECTS = "(})\\s*(\\{\"Item\":\\{\"" + PK0 + "\")";
```

Comment:
Regex is the sign of a bad life, one where data is badly formatted before it reaches you…